heareval/tasks/pipeline.py

```diff
@@ -154,6 +154,23 @@ def slugify_file_name(relative_path: str):
         """
         return f"{slugify(str(Path(relative_path).stem))}"
 
+    @staticmethod
+    def get_stratify_key(df: DataFrame) -> Series:
+        """
+        Get the stratify key
+
+        Subsampling is stratified based on this key.
+        Since hashing is only required for ordering the samples
+        for subsampling, the stratify key should not necessarily be a hash,
+        as it is only used to group the data points before subsampling.
+        The actual subsampling is done by the split key and
+        the subsample key.
+
+        By default, the label is used for stratification
+        """
+        assert "label" in df, "label column not found in the dataframe"
+        return df["label"].apply(str).apply(filename_to_int_hash)
```

Comment:
a) isn't this already a string? Why do we need to apply str?
b) Why are we hashing this?

Reply:
Will remove. Just kept for sanity. Also it need not be a hash as stated in the comments. Will change that as well